BackendNew/controllers/students.js

```diff
@@ -0,0 +1,27 @@
+const students = require("../models/students");
+
+exports.getStudents = (req, res) => {
+  var isSelected = req.query.isSelected;
+  var company = req.query.company;
+
+  if (isSelected) {
+    res.send(students.getSelectedStudents(isSelected));
```

Comment:
can you explain the logic behind this? Isn't this always directed to get selected students?

Reply:
The /students route has two query params. It will get either all students, selected students, not selected students or selected students by the company name. the first if statement compute if it asks for selected or not selected students. Second condition goes to check the company and third will get all students.

Reply:
Yes true. That is clear. But it not what i'm asking. if isSelected=true it will redirect to students.getSelectedStudents. But how to get not selected students?